rebasehelper/archive.py

```diff
@@ -190,42 +190,6 @@ def extract(cls, archive=None, filename=None, path=None, *args, **kwargs):
 
 
 @register_archive_type
-class SigArchiveType(ArchiveTypeBase):
-    """ .sig955015
-     files are not archives - this is a PGP file type"""
-    EXTENSION = ".sig"
-
-    @classmethod
-    def open(cls, filename=None):
-        pass
-
-    @classmethod
-    def extract(cls, archive=None, filename=None, path=None, *args, **kwargs):
-        if archive is not None:
-            raise RuntimeError("In Sig files, the archive (pos 1) argument is not used, but passed")
-        final_dir = os.path.join(path, os.path.basename(filename.rstrip(cls.EXTENSION)))
-        os.makedirs(final_dir)
-        shutil.copy(filename, final_dir)
-
-
-@register_archive_type
-class AscArchiveType(ArchiveTypeBase):
-    """ .asc files are not archives - this is a PGP file type"""
-    EXTENSION = ".asc"
-
-    @classmethod
-    def open(cls, filename=None):
-        pass
-
-    @classmethod
-    def extract(cls, archive=None, filename=None, path=None, *args, **kwargs):
-        if archive is not None:
-            raise RuntimeError("In Asc files, the archive (pos 1) argument is not used, but passed")
-        final_dir = os.path.join(path, os.path.basename(filename.rstrip(cls.EXTENSION)))
-        os.makedirs(final_dir)
-        shutil.copy(filename, final_dir)
-
-@register_archive_type
```

Comment:
Why did you remove it? Some files are needed from rebase-helper point of view. https://github.com/rebase-helper/rebase-helper/pull/117